print.c

```diff
@@ -42,3 +44,14 @@ int print(enum msglevel level, const char *fmt, ...)
 	}
 	return 0;
 }
+
+void dump_packet(uint8_t *packet, int size)
```

Comment:
Function definitions should return 'int' for error codes, even if the function can't possibly produce an error.